packages/contracts/contracts/L2/messaging/L2StandardERC721Factory.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/* Contract Imports */
+import { L2StandardERC721 } from "../../standards/L2StandardERC721.sol";
+
+/**
+ * @title L2StandardERC721Factory
+ * @dev Factory contract for creating standard L2 ERC721 representations of L1 ERC721s
+ * compatible with and working on the NFT bridge.
+ */
+contract L2StandardERC721Factory {
+    event StandardL2ERC721Created(address indexed _l1Token, address indexed _l2Token);
+
+    // Address of the L2 ERC721 Bridge. It currently exists only on Optimistic Kovan.
+    address public constant L2_ERC721_BRIDGE = 0xA779A0cA89556A9dffD47527F0aad1c2e0d66e46;
+
+    // Maps an L2 ERC721 token address to a boolean that returns true if the token was created
+    // with the L2StandardERC721Factory.
+    mapping(address => bool) public isStandardERC721;
```

Comment:
What is this mapping for exactly? Because developers can deploy custom L2 ERC721s that implement the correct interface and not be included in this mapping. Is is supposed to signify a "token list"?

Reply:
The purpose is just to give frontends an easy way to determine if an L2 ERC721 is a Standard ERC721. In other words, it gives frontends an easy way to tell users, "this L2 ERC721 contract doesn't implement any custom (i.e. potentially dangerous) functionality". The mapping isn't supposed to signify a token list, since that can include custom ERC721s too.